test/integration/createEnrollmentsFile.test.js

```diff
@@ -0,0 +1,37 @@
+require('dotenv').config()
+const test = require('ava')
+const createCoursesFile = require('../../server/createCoursesFile')
+const createEnrollmentsFile = require('../../server/createEnrollmentsFile')
+const fs = require('fs')
+
+function removeId (line) {
+  return line.replace(/u\w{6}/i, 'uXXXXXX')
+}
+
+test.skip('Generate enrollments.csv files for 2017-1-3', async t => {
```

Comment:
Should this test be skipped? If so, should it still remain or be deleted?

Reply:
You can see the reason why is it skipped in the description of this PR. We can have a discussion on this

Reply:
Ok, read it now. In that case I say we remove it, let's start with the other tests and then we can see how they work first